File: src/main/java/com/ruoyi/project/monitor/operlog/domain/OperLog.java
package com.ruoyi.project.monitor.operlog.domain;

import java.util.Date;
import lombok.Data;

/**
 * 操作日志记录 oper_log
 * 
 * @author ruoyi
 */
@Data
public class OperLog
{
    /** 日志主键 */
    private Integer operId;
    /** 模块标题 */
    private String title;
    /** 功能请求 */
    private String action;
    /** 请求方法 */
    private String method;
    /** 来源渠道 */
    private String channel;
    /** 操作员名称 */
    private String loginName;
    /** 部门名称 */
    private String deptName;
    /** 请求url */
    private String operUrl;
    /** 操作地址 */
    private String operIp;
    /** 请求参数 */
    private String operParam;
    /** 状态0正常 1异常 */
    private int status;
    /** 错误消息 */
    private String errorMsg;
    /** 操作时间 */
    private Date operTime;
}

File: src/main/resources/static/ruoyi/monitor/operlog/operlog.js
var prefix = "/monitor/operlog"

$(function() {
		var columns = [{
            checkbox: true
        },
        {
            field: 'operId',
            title: '日志编号'
        },
        {
            field: 'title',
            title: '模块'
        },
        {
            field: 'action',
            title: '功能'
        },
        {
            field: 'loginName',
            title: '登录名称'
        },
        {
            field: 'deptName',
            title: '部门名称'
        },
        {
            field: 'operIp',
            title: '主机'
        },
        {
            field: 'status',
            title: '操作状态',
            align: 'center',
            formatter: function(value, row, index) {
                if (value == 0) {
                    return '<span class="label label-success">成功</span>';
                } else if (value == 1) {
                    return '<span class="label label-primary">异常</span>';
                }
            }
        },
        {
            field: 'operTime',
            title: '操作时间'
        },
        {
            title: '操作',
            align: 'center',
            formatter: function(value, row, index) {
                var msg = '<a class="btn btn-warning btn-sm" href="#" title="详细信息" onclick="detail(\'' + row.operId + '\')"><i class="fa fa-search"></i></a> ';
                return msg;
            }
        }];
	var url = prefix + "/list";
    initTable(columns, url);
});

/*操作日志-详细*/
function detail(id) {
    var url = prefix + '/detail/' + id;
    layer_show("操作日志详细", url, '800', '500');
}

// 批量删除
function batchRemove() {
	var rows = getIdSelections("operId");
	if (rows.length == 0) {
		layer.msg("请选择要删除的数据");
		return;
	}
	layer.confirm("确认要删除选中的" + rows.length + "条数据吗?",{icon: 3, title:'提示'},function(index){
		_ajax(prefix + '/batchRemove', { "ids": rows }, "post");
	});
}
